site/css/footer.scss

```diff
@@ -1,131 +1,183 @@
 .site-footer {
```

Comment:
I rewrote this from scratch so if you want to check this out I would just look at the new file without the diff.